cm/lib/CMReconstruction.cc

```diff
@@ -7,6 +7,7 @@
 #include <TF2.h>
 #include <TMath.h>
 #include <TRandom.h>
+#include "CmdLineConfig.hh"
```

Comment:
Group includes in groups (separated by empty line) in the following order:
standard libraries, root headers, other libraries, project headers.